test/fixtures/api/user.json

```diff
@@ -0,0 +1,19 @@
+{
+	"login": "bkeepers",
+	"name": "Bee Keepers",
```

Comment:
It it a real user data?

Reply:
no, all fake.  Probably a user named that in github though...

Reply:
Yep. But at least it is all public

https://github.com/bkeepers